ibm/service/database/resource_ibm_database.go

```diff
@@ -2837,6 +2875,30 @@ func validateGroupHostFlavor(groupId string, resourceName string, group *Group)
 	return nil
 }
 
+func validateMultitenantMemoryCpu(groupId string, resourceName string, resourceDefaults *Group, group *Group) error {
```

Comment:
Not sure what Go conventions are (just learning), but you might be able to put guard clauses in here to simplify the rest of your code. Like in the first line of this method:
```
if group.CPU != nil {
  return nil
}
```